webpack/scenes/ContentViews/Details/ComponentContentViews/ComponentEnvironments.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Label } from '@patternfly/react-core';
+
+const ComponentEnvironments = ({ environments }) => {
+  const envLabels = environments.map(env =>
+    <React.Fragment key={env.id}><Label color="purple" href={`/lifecycle_environments/${env.id}`}>{`${env.name}`}</Label><br /></React.Fragment>);
+  return envLabels;
+};
```

Comment:
I think this will be better if you don't assign it to a variable:
```suggestion
const ComponentEnvironments = ({ environments }) => {
  return environments.map(env =>
    (
      <React.Fragment key={env.id}>
        <Label color="purple" href={`/lifecycle_environments/${env.id}`}>{`${env.name}`}</Label>
        <br />
      </React.Fragment>
    ));
};
```